### PR TITLE
[MAD][DATA1018] ELENA JIMENEZ POYATO

### PR DESCRIPTION
**Pull request by elenajpp**
Me ha parecido un ejercico largo pero efectivo
Un saludo 
Elena

### CODE REVIEW

**Comment by elenajpp** [comment]
@Gobinde43 Belen a mi me aparece como subido correctamente, lo vuelvo a intentar

**Comment by Gobinde43** [comment]
Entra en el Link de la pull request todos los ficheros están sin contenido.
Salvo que hayas modificado algo hoy

File: module-1/lab-numpy/your-code/main.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+print(n)
+a = np.random.randint(n, size=(2,3,5))
+print (a)
+
+
+n = np.random.random (100000)
+a=np.random.choice(n, size= (2,3,5))
+print (a)
+
+#4. Print a.
```
Comment (Gobinde43) [inline module-1/lab-numpy/your-code/main.py]: You don't need to write again the assignation of a. It's just print (a), as you have defined this variable above.

File: module-1/lab-numpy/your-code/main.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+b= np.ones ((5,2,3))
+
+
+#6. Print b.
+print (b)
+
+
+#7. Do a and b have the same size? How do you prove that in Python code?
+
```
Comment (Gobinde43) [inline module-1/lab-numpy/your-code/main.py]: You need to prove in Python how do you do that?

File: module-1/lab-numpy/your-code/main.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+print (d)
+
+ #Tienen las mismas dimensiones
+
+#11. Print a and d. Notice the difference and relation of the two array in terms of the values? Explain.
+
+print (a)
+print (d)
+
+#son exactamente iguales ya que multiplicas 1 por el numero aleatorio
```
Comment (Gobinde43) [inline module-1/lab-numpy/your-code/main.py]: Which ones are equal? a and d are not equal , there is a difference between both that you must explain.

File: module-1/lab-numpy/your-code/main.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+"""
+#16. Populate the values in f. For each value in d, if it's larger than d_min but smaller than d_mean, assign 25 to the corresponding value in f.
+If a value in d is larger than d_mean but smaller than d_max, assign 75 to the corresponding value in f.
+If a value equals to d_mean, assign 50 to the corresponding value in f.
+Assign 0 to the corresponding value(s) in f for d_min in d.
+Assign 100 to the corresponding value(s) in f for d_max in d.
+In the end, f should have only the following values: 0, 25, 50, 75, and 100.
+Note: you don't have to use Numpy in this question.
+"""
+for x, block in enumerate(d):
```
Comment (Gobinde43) [inline module-1/lab-numpy/your-code/main.py]: This has been copied from other students. If you feel you don't understand, please ask Marc or me for help but do not copy other students approaches.

File: module-1/lab-numpy/your-code/main.py [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+        [ 75.,  75.,  75.,  75.,  75.],
+        [ 25.,  75.,   0.,  75.,  75.]]])
+"""
+print (d)
+print (f)
+
+
+
+"""
+#18. Bonus question: instead of using numbers (i.e. 0, 25, 50, 75, and 100), how to use string values
```
Comment (Gobinde43) [inline module-1/lab-numpy/your-code/main.py]: This is a bonus question, you don't need to complete if you don't want or if you don't feel comfortable with. It's better not solving the exercise that copy the approaches of other students. Please take this into account for further exercises.

**Pull request closed without merging**